tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py

```diff
@@ -0,0 +1,122 @@
+              required=True, validators=[]),
+        Field(name='DATE_OF_BIRTH', type='string', startIndex=20, endIndex=28,
+              required=True, validators=[]),
+        Field(name='SSN', type='string', startIndex=28, endIndex=37,
+              required=True, validators=[]),
+        Field(name='RACE_HISPANIC', type='number', startIndex=37, endIndex=38,
+              required=True, validators=[]),
+        Field(name='RACE_AMER_INDIAN', type='number', startIndex=38, endIndex=39,
+              required=True, validators=[]),
+        Field(name='RACE_ASIAN', type='number', startIndex=39, endIndex=40,
+              required=True, validators=[]),
+        Field(name='RACE_BLACK', type='number', startIndex=40, endIndex=41,
+              required=True, validators=[]),
+        Field(name='RACE_HAWAIIAN', type='number', startIndex=41, endIndex=42,
+              required=True, validators=[]),
+        Field(name='RACE_WHITE', type='number', startIndex=42, endIndex=43,
+              required=True, validators=[]),
+        Field(name='GENDER', type='number', startIndex=43, endIndex=44,
+              required=True, validators=[]),
+        Field(name='RECEIVE_NONSSI_BENEFITS', type='number', startIndex=44, endIndex=45,
+              required=True, validators=[]),
+        Field(name='RECEIVE_SSI', type='number', startIndex=45, endIndex=46,
+              required=True, validators=[]),
+        Field(name='RELATIONSHIP_HOH', type='number', startIndex=46, endIndex=48,
+              required=True, validators=[]),
+        Field(name='PARENT_MINOR_CHILD', type='number', startIndex=48, endIndex=49,
+              required=True, validators=[]),
+        Field(name='EDUCATION_LEVEL', type='number', startIndex=49, endIndex=51,
+              required=True, validators=[]),
+        Field(name='CITIZENSHIP_STATUS', type='number', startIndex=51, endIndex=52,
+              required=True, validators=[]),
+        Field(name='UNEARNED_SSI', type='number', startIndex=52, endIndex=56,
+              required=True, validators=[]),
+        Field(name='OTHER_UNEARNED_INCOME', type='number', startIndex=56, endIndex=60,
+              required=True, validators=[])
+    ]
+)
+
+second_part_schema = RowSchema(
+    model=SSP_M3,
+    quiet_preparser_errors=True,
+    preparsing_validators=[
+        # validators.hasLength(150),  # unreliable.
+        validators.notEmpty(start=60, end=101),
+    ],
+    postparsing_validators=[],
+    fields=[
+        Field(name='RecordType', type='string', startIndex=0, endIndex=2,
+              required=True, validators=[]),
+        Field(name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
+              required=True, validators=[]),
+        Field(name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
+              required=True, validators=[]),
+        # Field(name='FIPS_CODE', type='string', startIndex=8, endIndex=19, required=True, validators=[
+        # ]),
```

Comment:
Why is this commented out? I note it contains same indices as CASE_NUMBER.

Reply:
it was leftover from a bad copy/paste job :) i've removed it